src/Socket.php

```diff
@@ -36,16 +36,12 @@ public function __construct(LoopInterface $loop, $socket, Buffer $buffer = null)
 
     public function getLocalAddress()
     {
-        if ($this->socket !== false) {
-            return stream_socket_get_name($this->socket, false);
-        }
+        return $this->sanitizeAddress(@stream_socket_get_name($this->socket, false));
```

Comment:
Yes, my suggestion was not to remove `@` from the `stream_socket_get_name` call again. I've been just curious if there are any consequences to potentially call `stream_socket_get_name` with `false` (or anything which isn't a resource).